libp2p/Session.h

```diff
@@ -39,7 +39,7 @@ namespace dev
 namespace p2p
 {
 
-struct Node;
+struct PeerInfo;
 
 /**
  * @brief The Session class
```

Comment:
This is not a very helpful comment :) Please stick to changes and not old and unchanged code. Perhaps we can establish a time for reviewing legacy code however doing this in a PR is distracting and takes time away from both of us from what is important, which is code that's changed. If I took the time to change, DRY, and refactor unrelated code then the PR will have a lot of noise and it'll be difficult to determine what actually changed.